content/blog/2020/09/camel-quarkus-release-1.1.0/index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Camel Quarkus 1.1.0 Released"
+date: 2020-09-14
+authors: ["jamesnetherton"]
+categories: ["Releases", "Camel Quarkus"]
+preview: "Camel Quarkus 1.1.0 Released"
+---
+
+Apache Camel Quarkus 1.1.0 has been released!
+
+We continue to integrate with the latest Camel and [Quarkus](https://quarkus.io/) releases, whilst adding new features and fixing bugs.
+
+For a full overview of the changes see the [1.1.0 milestone details](https://github.com/apache/camel-quarkus/milestone/6?closed=1).
+
+Here are some of the highlights.
+
+## Major component upgrades
+
+* [Camel 3.5.0](https://camel.apache.org/blog/2020/09/Camel35-Whatsnew/)
+* Quarkus 1.8.0
+
+## New extensions
+
+Camel Quarkus now has extensions for almost all Camel components. Many new ones have been added in this release, initially with support for JVM mode only. 
```

Comment:
```suggestion
We have added a massive amount of 174 new extensions in this release. This is because we hurry up to become [the only and default runtime](http://camel.465427.n5.nabble.com/camel-k-switch-to-Quarkus-as-default-framework-for-integrations-td5869959.html) of our sibling [Camel K](https://camel.apache.org/camel-k/latest/). 

159 of the new extensions have initially support for JVM mode only.
```